### ISSUE TITLE
2023 07 20, Accessibility, Insufficient color contrast in fillable text fields

### ISSUE THREAD

**Issue by jillwillcox** [Block81838/Block81838#6]
Brief Description 
Insufficient color contrast in fillable text fields

Brief Description
Insufficient color contrast in fillable text fields

Screengrab/videos (insert file names here)
https://www.loom.com/share/9f5eb4582fd845078bde70d4bc42cb7c

Link
Website being tested: https://unemployment.oregon.gov/contact-us
Color Contrast Checker: https://accessibleweb.com/color-contrast-checker/

Environment
(Device > OS > Browser)
Computer > macOS 13.4.1 > Chrome 114

Type of Bug:  Functional, Accessibility or Content
Accessibility

WCAG Reference Information
1.4.3: Contrast (Minimum) https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

Steps to reproduce
1)	00:00, in separate private windows, launch the page and the color contrast checker
2)	00:47, enter something in the search bar but do not submit
a)	00:58, note that the “SMALL” text example is slightly larger than the text in the search bar
i)	This means that if small text contrast fails then this smaller text contrast will fail as well
3)	01:03, using the eyedropper tool of the “Foreground color”, grab the color of the text you entered in step 2 (should be a greyish teal)
4)	01:11, using the eyedropper tool of the “background color”, grab the color of the text field background (should be a plain white)
5)	01:16, BUG: this color combination fails the small text contrast test
a)	This particular instance is present on all other pages
6)	01:38, scroll down to the “Email Address” field at the bottom of the test site; enter something in the “Email Address” field
a)	Note that, again, the “SMALL” text example is slightly larger than the text in the search bar
i)	This means that if small text contrast fails then this smaller text contrast will fail as well
7)	01:44, repeat steps 3 and 4 with the “Email Address” field
a)	02:00, BUG: this color combination fails the small text contrast test

Desired Result
•	Meaningful content (especially text) and the backgrounds have sufficient contrast in color so that users can easily see the meaningful content

Actual Result
•	Text in Fillable fields do not have sufficient contrast from the background

Screengrab/videos (insert file names here)
https://www.loom.com/share/9f5eb4582fd845078bde70d4bc42cb7c

Link
Website being tested: https://unemployment.oregon.gov/contact-us
Color Contrast Checker: https://accessibleweb.com/color-contrast-checker/

Environment
(Device > OS > Browser)
Computer > macOS 13.4.1 > Chrome 114

Type of Bug:  Functional, Accessibility or Content
Accessibility

WCAG Reference Information
1.4.3: Contrast (Minimum) https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

Steps to reproduce
1)	00:00, in separate private windows, launch the page and the color contrast checker
2)	00:47, enter something in the search bar but do not submit
a)	00:58, note that the “SMALL” text example is slightly larger than the text in the search bar
i)	This means that if small text contrast fails then this smaller text contrast will fail as well
3)	01:03, using the eyedropper tool of the “Foreground color”, grab the color of the text you entered in step 2 (should be a greyish teal)
4)	01:11, using the eyedropper tool of the “background color”, grab the color of the text field background (should be a plain white)
5)	01:16, BUG: this color combination fails the small text contrast test
a)	This particular instance is present on all other pages
6)	01:38, scroll down to the “Email Address” field at the bottom of the test site; enter something in the “Email Address” field
a)	Note that, again, the “SMALL” text example is slightly larger than the text in the search bar
i)	This means that if small text contrast fails then this smaller text contrast will fail as well
7)	01:44, repeat steps 3 and 4 with the “Email Address” field
a)	02:00, BUG: this color combination fails the small text contrast test

Desired Result
•	Meaningful content (especially text) and the backgrounds have sufficient contrast in color so that users can easily see the meaningful content

Actual Result
•	Text in Fillable fields do not have sufficient contrast from the background
